### PR TITLE
unify readmefiles of modules (beginning)

### PR DESCRIPTION
**Pull request by dev7ch**
beginning of readme, changelog and upgrade  (needs review) #1632

### CODE REVIEW

File: modules/crawler/README.md [hunk 1]
```diff
@@ -4,18 +4,23 @@
 [![Build Status](https://travis-ci.org/luyadev/luya-module-crawler.svg?branch=master)](https://travis-ci.org/luyadev/luya-module-crawler)
```
Comment (nadar) [inline modules/crawler/README.md]: Missing logo

File: modules/errorapi/README.md [hunk 2]
```diff
@@ -1,21 +1,26 @@
-Error API Module
-=========
+# Error API Module
```
Comment (nadar) [inline modules/errorapi/README.md]: LOGO

File: modules/frontendgroup/README.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
-Frontend User Group Module
-==========================
+# Frontend User Group Module
```
Comment (nadar) [inline modules/frontendgroup/README.md]: LOGO

File: modules/frontendgroup/README.md [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+
 
 [![LUYA](https://img.shields.io/badge/Powered%20by-LUYA-brightgreen.svg)](https://luya.io)
```
Comment (nadar) [inline modules/frontendgroup/README.md]: Badges

File: modules/gallery/README.md [hunk 5]
```diff
@@ -10,13 +10,16 @@ The gallery module allows you create folders and collection and upload images to
```
Comment (nadar) [inline modules/gallery/README.md]: LOGO & BADGES

File: modules/news/README.md [hunk 6]
```diff
@@ -6,13 +6,15 @@ The news module will provided you a basic news system with categories and tags.
```
Comment (nadar) [inline modules/news/README.md]: LOGO & BADGES

File: modules/remoteadmin/README.md [hunk 7]
```diff
@@ -1,5 +1,4 @@
-REMOTE ADMIN
-=============
+# REMOTE Admin Module
```
Comment (nadar) [inline modules/remoteadmin/README.md]: LOGO & BADGES (Slack)

File: modules/styleguide/README.md [hunk 8]
```diff
@@ -7,12 +7,16 @@ The basic idea behind the stylegiude is to give you the ability to ensure an adv
```
Comment (nadar) [inline modules/styleguide/README.md]: LOGO & BADGES

File: modules/crawler/README.md (hunk 1 above)
Comment (dev7ch) [reply modules/crawler/README.md]: done

File: modules/frontendgroup/README.md (hunk 3 above)
Comment (dev7ch) [reply modules/frontendgroup/README.md]: done

**Comment by nadar** [comment]
Total Downloads](https://packagist.org/packages/luyadev/luya-module-gallery)

[Latest Stable Version](https://packagist.org/packages/luyadev/luya-module-gallery)

please add download and stable version badges.

**Pull request merged**